youtube-helpers.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import json
+from youtube_dl import YoutubeDL
+
+
+def youtube_search(search_str):
+    """
+    Function takes in a search string and returns a dict object containing
+    Youtube search results.
+    {
+      "kind": "youtube#searchResult",
+      "etag": etag,
+      "id": {
+        "kind": string,
+        "videoId": string,
+        "channelId": string,
+        "playlistId": string
+      },
+      "snippet": {
+        "publishedAt": datetime,
+        "channelId": string,
+        "title": string,
+        "description": string,
+        "thumbnails": {
+          (key): {
+            "url": string,
+            "width": unsigned integer,
+            "height": unsigned integer
+          }
+        },
+        "channelTitle": string,
+        "liveBroadcastContent": string
+      }
```

Comment:
This docstring can be simplified. If you want to have information on what the dict structure looks like, a link would probably be better: https://developers.google.com/youtube/v3/docs/search

While descriptive docstrings and/or comments are super helpful, _overly_ long docstrings/comments can actually hinder readability (especially as the file size grows larger).  I'd highly recommend just replacing this with the url I mentioned earlier (or whichever url you'd like).

Reply:
good comment, I updated